aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -29,39 +29,39 @@ To start the wizard - note, to run the wizard you must be logged in as an admini
 
 Once selected you should see the 'Configure Your Server Wizard' opening screen:
 
-![](aspnet-and-iis6/_static/image1.jpg)
+![Screenshot of the Windows Server 2003 'Configure Your Server Wizard' opening screen. The next button is highlighted.](aspnet-and-iis6/_static/image1.jpg)
 
 Click 'Next &gt;':
 
-![](aspnet-and-iis6/_static/image2.jpg)
+![Screenshot of the Windows Server 2003 'Configure Your Server Wizard' 'preliminary steps' screen. The next button is highlighted.](aspnet-and-iis6/_static/image2.jpg)
 
 Click 'Next &gt;'
 
-![](aspnet-and-iis6/_static/image3.jpg)
+![Screenshot of the Windows Server 2003 'Configure Your Server Wizard' 'server role' screen. The menu shows the 'Application server' option highlighted.](aspnet-and-iis6/_static/image3.jpg)
 
 On this screen you will need to select 'Application server (IIS, ASP.NET) as the options to configure.
 
 Click 'Next &gt;'.
 
-![](aspnet-and-iis6/_static/image4.jpg)
+![Screenshot of the Windows Server 2003 'Configure Your Server Wizard' 'Application Server Options' screen. The Enable A S P . NET box is checked.](aspnet-and-iis6/_static/image4.jpg)
```

Comment:
@Rick-Anderson you may be seeing this because the style guide I followed calls for spaces in acronyms which are pronounced letter-by-letter and not spaces for acronyms that are pronounced as a word ("Add spaces to acronyms where each character should be read (e.g., "H T M L") but not for words (e.g., "NASA")."). https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main 

I will go through now and make the changes your requested. Thank you for your comments!